CHANGELOG/CHANGELOG-2.10.md

```diff
@@ -0,0 +1,55 @@
+# Release notes for v2.10.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+
+# Changelog since v2.9.0
+
+## Changes by Kind
+
+### Failing Test
```

Comment:
Would this be better as uncategorized?

Reply:
It fixed a failed trivy CI job :-). I moved it to a bugfix, PTAL.